torch/_dynamo/config_utils.py

```diff
@@ -1,14 +1,15 @@
 import contextlib
 
 import copy
+import json
 import pickle
 import unittest
 from types import FunctionType, ModuleType
 from typing import Any, Dict, Set
 from unittest import mock
 
 # Types saved/loaded in configs
-CONFIG_TYPES = (int, float, bool, type(None), str, list, set, tuple, dict)
+CONFIG_TYPES = (int, float, bool, type(None), str, list, tuple, dict)
```

Comment:
The config types are in fact here, and I believe are in 1-1 correspondence with JSON serializability now.

Reply:
![Uploading image.png…]()